soft4pes/utils/conversions.py

```diff
@@ -30,24 +31,102 @@ def abc_2_alpha_beta(abc):
     return ab.T
 
 
+def alpha_beta_2_abc(alpha_beta):
+    """
+    Convert quantity from abc-frame to alpha-beta frame using inverse 
+    reduced Clarke transformation. The common mode component is neglected.
+
+    Parameters
+    ----------
+    abc : n x 3 ndarray of floats
+        Input quantity in abc-frame. 
+
+    Returns
+    -------
+    n x 2 ndarray of floats
+        Quantity in alpha-beta frame. 
+    """
+
+    n = len(alpha_beta)
+    abc = np.zeros((n, 3))
+    K_inv = np.array([[1, 0], \
+                      [-1 / 2, np.sqrt(3) / 2],
+                      [-1 / 2, -np.sqrt(3) / 2]])
+
+    abc = np.dot(K_inv, alpha_beta.T)
```

Comment:
Is this .T required? I think that .T does not transpose alpha_beta as it is a 1D array...

Reply:
This was needed to do the transformation for `n` samples. I removed it now, as it can cause confusion. This will be later implemented with a for-loop, so that the implementation of conversion for `n` samples will be the same for all functions